examples/example_02_matlab_function/treeOptData/DoeData.csv

```diff
@@ -0,0 +1,10 @@
+4.246952912825710413e+00
```

Comment:
I wrote some lines which "safes" the data points of the metamodel. In the future, I plan to implement a function to read these files. This would open the possibility to load an existing metamodel and perform analyzes, without rerunning all the simulations.

But you are correct. I will edit the .qitignore File in order to not track these files.